### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Rockyraven**
created home page
add component 
install mockbee install

### CODE REVIEW

**Review by arjun-computer-geek — APPROVED** [review]
All are good as usual. Go ahead.

File: src/index.css [hunk 1]
```diff
@@ -5,4 +5,5 @@ body{
     margin: 0px;
     padding: 0px;
```
Comment (Developers-codz) [inline src/index.css:6]: 0 only will work??

File: src/pages/homepage/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Sidebaar } from '../../component'
```
Comment (Developers-codz) [inline src/pages/homepage/HomePage.js:2]: suggestion: can use absolute path

**Review by Developers-codz — APPROVED** [review]
Looks good to go, very small PR found nothing much to suggest

**Pull request merged**